### ISSUE TITLE
Providers pod spec doesn't have Core as a dependency

### ISSUE THREAD

**Issue by welshm** [zendesk/zendesk_sdk_ios#385]
- SDK version:
2.0.3

- Xcode version:
9.3.1

- iOS or Simulator:
Not applicable

- iOS Version:
Not applicable

- Swift version:
Objective-C

Steps to Reproduce:

Use pod 'ZendeskSDK/Providers', '2.0.3' in your podfile.

Can't use the SDK unless I also add 
pod 'ZendeskSDK/Core', '2.0.3'

**Comment by KillianSmith0** [comment]
Hi @welshm, thanks for reaching out.

Our `ZendeskSDK/Providers` pod depends on the `ZendeskSDK/Core` pod. Core contains Authentication and instantiation of the SDK. Without it the providers won't be able to access your instance of Zendesk. 

I'm going to close this ticket as it's not an issue, any further questions, feel free to open it again.

**Comment by welshm** [comment]
```
s.subspec 'Providers' do |ss|
    ss.platform     = :ios, '9.0'
    ss.ios.vendored_frameworks = 'ZendeskSDK/4.1/ZendeskProviderSDK.framework'
    ss.preserve_paths = 'ZendeskSDK/4.1/ZendeskProviderSDK.framework'
end
```


Does not contain

`ss.dependency 'ZendeskSDK/Core'`

**Comment by KillianSmith0** [comment]
Hi @welshm, thanks for getting back to us, and finding this issue, we'll make the required changes and get back to you as soon as.

**Comment by welshm** [comment]
@KillianSmith0 No worries! Thanks for taking a look at this. If there's a reason that it can't be a dependency, that's fine too. Just took me a while scratching my head trying to figure out why it didn't work (as it will fail if you just follow the SDK installation instructions provided on the Zendesk website - which say you can just use the Providers subspec)

**Comment by KillianSmith0** [comment]
@welshm 
No problem, you're right it should be a dependency, it was a good spot! We've updated the podspec in the repo. 
However, Cocoapods will not allow us to update the Pod without bumping the version number. We are currently working on a patch for the SDK, and the fix for this will be included. Unfortunately that means you will have to include the `Core` pod also. Hopefully this doesn't cause a major inconvenience, and as soon as the fix is up, we'll let you know! Thanks again for finding the issue

**Comment by divo** [comment]
@welshm we have just released 2.0.4. The provider subspec should work correctly now. Thanks for reporting this.

**Comment by welshm** [comment]
Should this be re-opened since the release was reverted?

**Comment by KillianSmith0** [comment]
We've just released to 2.0.5, the pod should be available to update in the next hour or so, thanks for your patience
